CONTRIBUTING.md

```diff
@@ -182,6 +182,98 @@ components/
+  });
+};
+
+describe("Components/My Custom Element", () => {
+  it("should do something expected", () => {
+    expect("something").equal("some" + "thing");
+  });
+});
+```
+
+#### Testing the Component
+
+With the spec file foundation, the custom element can now be created and applied to the mock DOM environment. To do this, we use the Web Test Runner (WTR) framework which provides the scaffolding to test Web Componets in a browser environment.
+
+The WTR and the full setup is covered in greater detail within [the GreenwoodJS test runner docs](https://super-tapioca-5987ce.netlify.app/guides/ecosystem/web-test-runner/). In an attempt to not duplicate those docs, an outline has been provided here, but please refer to those docs for a full user guide.
+
+1. **[Setup](https://super-tapioca-5987ce.netlify.app/guides/ecosystem/web-test-runner/#setup)**
+
+   Install dependencies and configure WTR. This project has that covered already! ✅ (Let's chat if there becomes a need to reconfigure the current setup.)
+
+1. **[Usage](https://super-tapioca-5987ce.netlify.app/guides/ecosystem/web-test-runner/#usage)**
+
+   Within a traditional `before` block, create the custom element, and append to the DOM. See the example below. 👇
+
+1. **[Static Assets](https://super-tapioca-5987ce.netlify.app/guides/ecosystem/web-test-runner/#static-assets)**
+
+   If experiencing a 404 for a missing asset, a simple middleware can help mock that request. Otherwise, you can skip this section.
+
+1. **[Install Resource Plugins](https://super-tapioca-5987ce.netlify.app/guides/ecosystem/web-test-runner/#resource-plugins)**
+
+   If using a Greenwood resource plugin, you'll need to provide that info to stub out the signiture.
+
+1. **[Mock Data Requests](https://super-tapioca-5987ce.netlify.app/guides/ecosystem/web-test-runner/#content-as-data)**
+
+   If using Greenwood's Content as Data feature, mocking `fetch` with mock data is necessary.
+
+#### Examples
+
+With the test setup complete, we can now create the block necessary to put the custom element under test.
+
+Populating the previous example, we include the `before` block and create, customtize, and append the element using the Browser API.
+
+```js
+import { expect } from "@esm-bundle/chai";
+import "./my-custom-element.js";
+
+// ...mocks truncated....
+
+describe("Components/My Custom Element", () => {
+  let myElement;
+
+  before(async () => {
+    // create.
+    myElement = document.createElement("app-my-element"); // the tagname as it appears in the browser
+
+    // customize.
+    myElement.addAttribute("route", "/some/path/to/a/page");
+
+    // append.
+    document.body.appendChild(myElement);
+
+    // all ready.
+    await myElement.updateComplete;
+  });
+
+  after(() => {
+    // cleanup.
+    myElement.remove();
+    myElement = null;
+  });
+
+  it("should do something expected", () => {
+    expect("something").equal("some" + "thing");
+  });
+
+  // ...it()...
+});
+```
+
 ### Storybook
```

Comment:
I think it might be good add a call out re: the content as data stuff here in the Storybook docs, just to be complete. 

Here is an example for reference
https://github.com/ProjectEvergreen/www.greenwoodjs.dev/blob/main/src/components/blog-posts-list/blog-posts-list.stories.js

And the relevant website docs
https://super-tapioca-5987ce.netlify.app/guides/ecosystem/storybook/#content-as-data